<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconLibDlg</class>
 <widget class="QDialog" name="IconLibDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор иконки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/iconlib.qrc">
    <normaloff>:/iconlib/image.png</normaloff>:/iconlib/image.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="aliasLabel">
     <property name="text">
      <string>Элемент: </string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="textElideMode">
      <enum>Qt::TextElideMode::ElideMiddle</enum>
     </property>
     <property name="movement">
      <enum>QListView::Movement::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::ResizeMode::Fixed</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::LayoutMode::SinglePass</enum>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="gridSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::ViewMode::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/iconlib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IconLibDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IconLibDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
